libkbfs/ops.go

```diff
@@ -1252,6 +1252,8 @@ func invertOpForLocalNotifications(oldOp op) (newOp op, err error) {
 		}
 	case *GCOp:
 		newOp = op
+	case *resolutionOp:
```

Comment:
When unstaging changes, we invert every op on the unmerged branch.  Now a `resolutionOp` is included in every sync operation, and so the test we have for checking unstaging was running into this and failing here.